scripts/run-swift-format.sh

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+##===----------------------------------------------------------------------===##
+##
+## This source file is part of the SwiftASN1 open source project
+##
+## Copyright (c) 2022-2023 Apple Inc. and the SwiftASN1 project authors
+## Licensed under Apache License v2.0
+##
+## See LICENSE.txt for license information
+## See CONTRIBUTORS.txt for the list of SwiftASN1 project authors
+##
+## SPDX-License-Identifier: Apache-2.0
+##
+##===----------------------------------------------------------------------===##
+##===----------------------------------------------------------------------===##
+##
+## This source file is part of the SwiftOpenAPIGenerator open source project
+##
+## Copyright (c) 2023 Apple Inc. and the SwiftOpenAPIGenerator project authors
+## Licensed under Apache License v2.0
+##
+## See LICENSE.txt for license information
+## See CONTRIBUTORS.txt for the list of SwiftOpenAPIGenerator project authors
+##
+## SPDX-License-Identifier: Apache-2.0
+##
+##===----------------------------------------------------------------------===##
+set -euo pipefail
+
+log() { printf -- "** %s\n" "$*" >&2; }
+error() { printf -- "** ERROR: %s\n" "$*" >&2; }
+fatal() { error "$@"; exit 1; }
+
+CURRENT_SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
```

Comment:
here too, `lower_snake_case` everywhere unless you're using a special variable like `$PATH` or other things that are meant to influence other processes (e.g. `HTTP_PROXY` and what not)

Reply:
I don't want to modify this script as it is copy and pasted over from https://github.com/apple/swift-openapi-generator/blob/main/scripts/run-swift-format.sh

Reply:
:'( I'd rather not proliferate poor shell scripting practises. Maybe fix upstream?